src/extension/interop/ExtensionInterop.cpp

```diff
@@ -34,6 +37,11 @@ void DebugDrawerAddLine(DebugDrawer* drawerInstance, JVecF3* from, JVecF3* to, J
             *reinterpret_cast<godot::Color*>(colour));
 }
 
+bool Unwrap(float texelSize, ArrayMesh* mesh)
+{
+	return Thrive::Unwrap(texelSize, *reinterpret_cast<godot::ArrayMesh*>(mesh));
```

Comment:
I'm pretty sure this is a tab character which clang-format shouldn't put according to our configured style.